static/app/components/panels/panelHeader.tsx

```diff
@@ -41,3 +42,21 @@ const PanelHeader = styled('div')<Props>`
 `;
 
 export default PanelHeader;
+
+const ClickablePanelHeader = styled(PanelHeader)`
+  user-select: none;
+  cursor: pointer;
+`;
+
+export function CollapsePanelHeader(props: {
+  children: React.ReactNode;
+  collapsed?: boolean;
+  onClick?: () => void;
+}) {
+  return (
+    <ClickablePanelHeader onClick={props.onClick}>
+      {props.children}
+      <IconChevron direction={props.collapsed ? 'right' : 'down'} />
+    </ClickablePanelHeader>
+  );
+}
```

Comment:
We should remove the `PanelHeader`'s bottom border when `collapsed` is true
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/44172267/181372446-ce915e03-3245-41b5-880c-1e3102a46278.png">